lib/TPP/PassBundles/LocalDialectsLowering.cpp

```diff
@@ -58,7 +58,6 @@ struct LocalDialectsLowering
   void constructPipeline() override {
     pm.addNestedPass<func::FuncOp>(createConvertCheckToLoops());
     pm.addNestedPass<func::FuncOp>(createConvertPerfToLoops());
-    pm.addPass(createConvertXsmmToFunc());
     pm.addPass(createConvertPerfToFunc());
```

Comment:
Why do you remove this here but add it in the test?

Reply:
Added a comment in the file.